frb_codegen/src/lib.rs

```diff
@@ -40,10 +40,22 @@ mod parser;
 mod source_graph;
 mod target;
 mod transformer;
-mod utils;
+pub mod utils;
 use error::*;
 
 pub fn frb_codegen(config: &config::Opts, all_symbols: &[String]) -> anyhow::Result<()> {
+    frb_codegen_multi(config, &[config.clone()], 0, all_symbols)
+}
+
+/// the `all_configs` here is used only for multi-blocks
+pub fn frb_codegen_multi(
+    config: &config::Opts,
+    all_configs: &[config::Opts],
+    index: usize,
```

Comment:
uber-nit: Given that config == all_configs[index], what about removing `config` parameter

(to be honest it has been some time after I first reviewed this PR, so my memory is not that fresh. hopefully this uber-nit is correct!)

Reply:
Refined in https://github.com/fzyzcjy/flutter_rust_bridge/pull/1024/commits/1217983475c2a047aa56593d4bb47da14ee354c1.